### ISSUE TITLE
Failing to bind

### ISSUE THREAD

**Issue by bilal-fazlani** [mkroli/dns4s#10]
I am getting an error while trying to start the DNS

```
akka.pattern.AskTimeoutException: Ask timed out on 
[Actor[akka://DnsServer/user/dns4s#-1020510439]] after [5000 ms]. Message of type 
[com.github.mkroli.dns4s.akka.Dns$Bind]. A typical reason for `AskTimeoutException` is that 
the recipient actor didn't send a reply.
```
This is my actor:

```scala
class SampleActor extends Actor {
  override def receive: Receive = {
    case Bound =>
      println(s"Bound message received") // <- this get's printed
      sender() ! "OK"
    case x =>
      println(x)
      sender() ! "OK"
  }
}

object SampleActor {
  def start: Future[Any] = {
    implicit val system: ActorSystem = ActorSystem("DnsServer")
    implicit val timeout: Timeout    = Timeout(5 seconds)
    val f                            = IO(Dns) ? Dns.Bind(system.actorOf(Props[SampleActor]), 6345)
    f.onComplete {
      case Failure(exception) =>
        println("DNS service failed to start")
        println(exception.getClass.getSimpleName)
        println(exception.getMessage)
        exception.printStackTrace()
        sys.exit(1)
      case Success(value) =>
        println(s"DNS service: $value")
    }
    f
  }
}

```

**Comment by bilal-fazlani** [comment]
sbt file:

```sbt
name := "dns4s_try"

version := "0.1"

scalaVersion := "2.12.8"

resolvers += "bintray" at "http://jcenter.bintray.com"

libraryDependencies ++= Seq(
  "com.typesafe.akka" %% "akka-actor" % "2.4.12",
  "com.github.mkroli" %% "dns4s-akka" % "0.13.0"
)
```

**Comment by mkroli** [comment]
Thanks for the report. Should be fixed on master (see 6c1f19a528482899b3989be44d496c846f78c33a). Could you verify?

**Comment by bilal-fazlani** [comment]
Thanks for the quick response! I am not if I can verify this without a release. But if you have tested it, then I am pretty eager to give it a try. Would you be able to make a release ? @mkroli

**Comment by mkroli** [comment]
It's released in 0.14.0.

**Comment by bilal-fazlani** [comment]
It works! thanks :)
